go.mod

```diff
@@ -1,67 +1,128 @@
 module github.com/phoreproject/synapse
 
-go 1.13
+go 1.18
```

Comment:
wazero won't work before 1.18, and I noticed CI is using 1.19, so figured maybe safe to update this.